md/customize-living-application-layout.md

```diff
@@ -112,3 +112,30 @@ Those variables are responsible of the SEVERE error logs on server.
 7. On Bonita Portal server edit the layout and import the newly exported layout  
 8. confirm all the messages  
 9. Validate that your application has a layout that fits your requirements. 
+
+### Ui-bootstrap.js library removed from runtime
+In our first design iteration, forms, pages and layouts designed with the UI Designer embedded 
+[UI Bootstrap js, version 0.13.4](http://angular-ui.github.io/bootstrap/versioned-docs/0.13.4/) by default and silently, even when not needed. 
+This issue has been fixed in the 7.5.0 version, we have removed it so you can use it only when you need it, and with the version you choose. 
```

Comment:
in version 7.5.0
, we removed it so you can embed it as an asset ponly when you need it, an in the version of your choice.